lib/webdrivers/chromedriver.rb

```diff
@@ -87,6 +87,14 @@ def file_name
         System.platform == 'win' || System.wsl? ? 'chromedriver.exe' : 'chromedriver'
       end
 
+      def apple_m1_cpu?
+        RUBY_PLATFORM.include?('arm64-darwin')
+      end
+
+      def apple_m1_compatible?(version)
```

Comment:
How about we combine `apple_m1_cpu?` and `apple_m1_compatible ?` as one `apple_m1_compatible?`? Then line 107 can simply be:

```ruby
apple_arch = apple_m1_compatible?(version) ? '_m1' : ''
```

I am not seeing a use case where we would ever use these methods separately. Do you?